### PR TITLE
converted [nodeping npm nexus node] from static classes to props

### PR DESCRIPTION
**Pull request by jabbar86**

### CODE REVIEW

File: services/node/node-base.js
Comment (calebcartwright) [inline services/node/node-base.js:10]: the LTS and Current services have different routes, so they need to define their respective routes. Can't hardcode this in the parent class here

Comment (jabbar86) [reply services/node/node-base.js]: @calebcartwright  yes trying that.its in review phase

Comment (calebcartwright) [reply services/node/node-base.js]: Note that you can mark a PR as a draft if you aren't ready for a review, but otherwise as maintainers we'll review and provide feedback on open PRs especially updates to existing ones.

If you're just trying to get the tests to run, also note that you can run them locally `npm run test:services -- --only=node,nexus` in this case

Comment (jabbar86) [reply services/node/node-base.js]: @calebcartwright  wohooo it worked. Don't merge it pushing other modules

**Comment by calebcartwright** [comment]
The npm and nodeping changes look fine, but there's no longer anything in here for `node` nor for `nexus`. Was this intentional?

**Comment by jabbar86** [comment]
> 
> 
> The npm and nodeping changes look fine, but there's no longer anything in here for `node` nor for `nexus`. Was this intentional?

Hey, @calebcartwright you can see the changes of node and nexus in the file changed.

File: services/node/node-current.service.js
Comment (calebcartwright) [inline services/node/node-current.service.js:8]: Please remove this

File: services/node/node-lts.service.js
Comment (calebcartwright) [inline services/node/node-lts.service.js:8]: Please remove

File: services/node/node-base.js
Comment (calebcartwright) [inline services/node/node-base.js:10]: Please remove

**Review by calebcartwright — COMMENTED** [review]
Almost there!

**Comment by calebcartwright** [comment]
> you can see the changes of node and nexus in the file changed.

Yup, must have had a filter on earlier when I only saw the 9 files :man_shrugging:

File: services/node/node-base.js
Comment (jabbar86) [reply services/node/node-base.js]: @calebcartwright all the comments are resolved.

**Review by calebcartwright — APPROVED** [review]
Thanks!

**Pull request merged**
